plane_mcp/__main__.py

```
@@ -9,13 +9,38 @@
+
+    def filter(self, record: logging.LogRecord) -> bool:
+        user_id = None
+        display_name = None
+        try:
+            token = get_access_token()
+            if token:
+                user_id = token.claims.get("sub")
+                display_name = token.claims.get("display_name")
+        except Exception:
+            # Never let logging enrichment break a request.
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)